src/components/CalculateVolumeView/ProcessPickerResults.js

```diff
@@ -0,0 +1,23 @@
+import {InnerProcessList} from "../List/ProcessList/InnerProcessList";
+import {InvalidFormMessage, ProcessPickerContainer} from "./Styles";
+import React from "react";
+import {SectionHeader} from "../Detail/SectionHeader";
+
+export const ProcessPickerResults = ({processes, setProcess}) => {
+  return (
+    <>
+      <SectionHeader>Resultados de la búsqueda</SectionHeader>
```

Comment:
Creo que quedaria mejor si este titulo solo aparece despues de que efectivamente se realiza una busqueda. Actualmente esta bajo cualquier circunstancia

Reply:
Are you sure?

Reply:
shit